### ISSUE TITLE
Can you please update this example for latest version of library?

### ISSUE THREAD

**Issue by Cheesy008** [stalniy/casl-nuxt-example#8]
I faced with some problems when trying to integrate current version of casl with nuxt. It would be great if this app example was updated

**Comment by andorfermichael** [comment]
Yes, please provide an example for nuxt3, I would be really thankful for that.
